content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -150,20 +150,21 @@ ui.properties:          83 bytes
 
 Use the option `--from-env-file` to create a ConfigMap from an env-file, for example:
 ```shell
+wget https://k8s.io/docs/tasks/configure-pod-container/configmap/kubectl/game-env-file.properties -O configure-pod-container/configmap/kubectl/game-env-file.properties
+cat configure-pod-container/configmap/kubectl/game-env-file.properties
```

Comment:
We could also (in this PR, or separately) make the commands and the sample output separate, so as to match the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output).